nixos/modules/programs/localsend.nix

```
@@ -0,0 +1,27 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.programs.localsend;
+  firewallPort = 53317;
+in {
+  options.programs.localsend = {
+    enable = mkEnableOption (mdDoc "localsend");
```

```suggestion
    enable = mkEnableOption "localsend, an open source cross-platform alternative to AirDrop";
```

note that `mdDoc` is removed. You should remove it altogether (description is mdDoc by default now)

---

Removed.